third_party/google/protobuf/descriptor.proto

```
@@ -0,0 +1,1434 @@
+
+// Author: kenton@google.com (Kenton Varda)
+//  Based on original Protocol Buffers design by
+//  Sanjay Ghemawat, Jeff Dean, and others.
+//
+// The messages in this file describe the definitions found in .proto files.
+// A valid .proto file can be translated directly to a FileDescriptorProto
+// without any other information (e.g. without reading its imports).
+
+syntax = "proto2";
+
+package google.protobuf;
```

Buf (1.72.0)

[error] Package name "google.protobuf" should be suffixed with a correctly formed version, such as "google.protobuf.v1".

(PACKAGE_VERSION_SUFFIX)